### ISSUE TITLE
[Enhancement]: SMTP sunsetting

### ISSUE THREAD

**Issue by amasterengineer** [advplyr/audiobookshelf#3517]
### Type of Enhancement

Server Backend

### Describe the Feature/Enhancement

Many email providers are sunsetting basic authentication to include smtp. Will this application be updated to allow email notifications via moderen authentication, that outlook and Gmail will be supporting after fall 2024

### Why would this be helpful?

Many email providers are sunsetting basic authentication to include smtp. Will this application be updated to allow email notifications via moderen authentication, that outlook and Gmail will be supporting after fall 2024

### Future Implementation (Screenshot)

Many email providers are sunsetting basic authentication to include smtp. Will this application be updated to allow email notifications via moderen authentication, that outlook and Gmail will be supporting after fall 2024

### Audiobookshelf Server Version

2.15.0

**Comment by nichwall** [comment]
Where do you see that SMTP access is going away? The only thing I can find online is that Google removed using a plain username/password in 2022 and that you need to use an app password, which ABS supports.

To set up an app password for Google, you can reference their help article.
https://support.google.com/mail/answer/185833?hl=en

**Comment by nichwall** [comment]
Ah, thanks for the links.

I'm not sure on the time table for implementing OAuth for emails. There is OAuth for authenticating users in ABS, but not sure if that is usable for authenticating for email or needs a completely separate system.

One thing to note is the Google article specifically says app passwords will still continue to work (but still not as good as OAuth).

**Comment by ponchohoncho** [comment]
A dedicated SMTP provider is a good idea. The free tiers of most SMTP providers should get you along just fine. smtp2go is my favorite. More reasons here: https://smtp.rodeo/

**Comment by devnoname120** [comment]
I agree with @ponchohoncho. Supporting each provider's convoluted auth flow is not worth it IMO. And it's a security risk as well because if your Audiobookshelf instance gets hacked then your email address will get impersonated.

**Comment by PandinoSix13** [comment]
Hi everyone, I'm desperate, I can't find the function in ABS that accepts oauth2... what do you think, will something be implemented in the following days? Thank you

**Comment by ZLoth** [comment]
https://support.google.com/a/answer/176600?hl=en:

>Starting May 1, 2025, Google Workspace accounts no longer support [less secure apps](https://support.google.com/accounts/answer/6010255), third-party apps, or devices that ask you to sign in to your Google Account with your username and password. You must use OAuth to let these apps and devices access your account. Third-party email apps that are no longer supported include Microsoft Outlook and the mail app on iOS and MacOS. For detailed instructions and information, visit [Transition from less secure apps to OAuth](https://support.google.com/a/answer/14114704). For the latest dates, visit [Google Workspace Updates](https://workspaceupdates.googleblog.com/2023/09/winding-down-google-sync-and-less-secure-apps-support.html).

**Comment by scj643** [comment]
I use https://support.google.com/a/answer/2956491 and have a system with a dedicated IP setup for smtp-relay
